### DISCUSSION TITLE
Education application issues

### DISCUSSION THREAD

**Discussion by zsotroav** [community/community#54235] [GitHub Education]
# To GitHub education staff,

I was directed to post a help request here by GitHub support, if you are not staff with insight into the system, please ignore this, you are not going to be able to help in any way. ~~Why am I forced to post such a private request publically anyways?~~

The GitHub education application automatically refused my application with the official state-issued photo student ID due to "it being from a third party". Said third party is the official government entity tasked with managing schools. In its current state, it is impossible for me to apply for the student pack due to the most official document not being official enough. Furthermore, the application form only allows for the taking of one picture and the ID has important details on both sides, making meeting the application requirements impossible. (Name is on one side, expiry date and school on the other)

### A suggestion...(optional)

A private support system? It is unacceptable that I have to post a private support request publicly.

**Comment by trimi3** [comment]
did you solve your problem? I'm facing similar issue.

**Comment by zsotroav** [comment]
bump
